### DISCUSSION TITLE
The given role or permission should use guard `` instead of `Web`？

### DISCUSSION THREAD

**Discussion by suzhiF** [spatie/laravel-permission#2245] [Q&A] [answered]
The given role or permission should use guard `` instead of `Web`？

**Comment by suzhiF** [comment]
~~~
class User extends Authenticatable
{
    use HasRoles;

    protected $guard_name = 'web';
}
~~~

文档中并没有这一句 `protected $guard_name = 'web'` 
![image](https://user-images.githubusercontent.com/46362400/201851645-0807adc4-5b37-4af4-a5c4-e81ebe7a5974.png)

**Comment by erikn69** [comment, marked as answer]
Dupicate of
- https://github.com/spatie/laravel-permission/discussions/2108
- https://github.com/spatie/laravel-permission/discussions/2242
